mtv_dl.py

```diff
@@ -464,7 +464,11 @@ def _show_hash(channel: str, topic: str, title: str, size: int, start: datetime)
         h.update(topic.encode())
         h.update(title.encode())
         h.update(str(size).encode())
-        h.update(str(start.timestamp()).encode())
+        try:
+            h.update(str(start.timestamp()).encode())
```

Comment:
I think this is an acceptable solution. The hash will be different on the platform, but that's actually fine as because it stays stable for the history at least. After all, my decision to use the timestamp here was not optimal (ISO string would maybe work better), but it's also not so important.